src/Persistence/Sql.php

```diff
@@ -434,9 +436,7 @@ public function tryLoad(Model $model, $id): ?array
      */
     public function load(Model $model, $id): array
     {
-        $data = $this->tryLoad($model, $id);
-
-        if (!$data) {
+        if (!$data = $this->tryLoad($model, $id)) {
```

Comment:
please, do not assign in separate lines, also do not assign more variables per one line

Reply:
Reverted